### PR TITLE
Fix #11 - Clean up existing module metrics

### LINKED ISSUE #11: Clean up existing module metrics
These module infos have been removed (not emitted). However, the documentation is forgotten to cleaned up.

_Originally posted by @joehu21 in https://github.com/valkey-io/valkeyJSON/pull/1#discussion_r1859319898_

### PR DESCRIPTION
**Pull request by joehu21**
Fix #11 - Clean up existing module metrics

### CODE REVIEW

**Review by roshkhatri — APPROVED** [review]

**Review by roshkhatri — APPROVED** [review]
LGTM

File: src/json/stats.h [hunk 1]
```diff
@@ -1,30 +1,12 @@
...
- *    json_update_histogram: dynamic histogram for update operations (JSON.SET, JSON.STRAPPEND and
- *        JSON.ARRAPPEND). Value of the i_th element is number of update operations with input JSON size falling into
- *        bucket i.
- *    json_delete_histogram: dynamic histogram for delete operations (JSON.DEL, JSON.FORGET, JSON.ARRPOP and
- *        JSON.ARRTRIM). Value of the i_th element is number of delete operations with deleted values' size falling
- *        into bucket i.
- *
- * Histogram buckets:
- *    [0,256), [256,1k), [1k,4k), [4k,16k), [16k,64k), [64k,256k), [256k,1m), [1m,4m), [4m,16m), [16m,64m), [64m,INF).
- *    Each bucket represents a JSON size range in bytes.
```
Comment (roshkhatri) [inline src/json/stats.h:23]: Do we also want to remove the histogram section from https://github.com/valkey-io/valkeyJSON/blob/373a92e5d92eef764b65cf1ca5a8a0a5518fafee/src/json/json.cc#L2231

Comment (joehu21) [reply src/json/stats.h:23]: Will remove the info sections.

Comment (joehu21) [reply src/json/stats.h:23]: removed

**Review by roshkhatri — APPROVED** [review]

**Pull request merged**